cloud-controller-manager/do/cloud.go

```diff
@@ -66,6 +68,12 @@ func newCloud() (cloudprovider.Interface, error) {
 		opts = append(opts, godo.SetBaseURL(overrideURL))
 	}
 
+	if version != "" {
+		opts = append(opts, godo.SetUserAgent("digitalocean-cloud-controller-manager"+"/"+version))
+	} else {
+		opts = append(opts, godo.SetUserAgent("digitalocean-cloud-controller-manager"+"/dev"))
```

Comment:
I think the new section could be slightly simplified by setting `version` appropriately and doing a single append call only, i.e.:

```go
if version == "" {
  version = "dev"
}
opts = append(opts, godo.SetUserAgent("digitalocean-cloud-controller-manager"+"/"+version))
```

Reply:
:ok_hand: I like it. Updated.